File: baekjoon_all/03000+/boj_3181.cpp
// Solve 2024-02-12

#include <bits/stdc++.h>
using namespace std;

#define FASTIO ios_base::sync_with_stdio(false);cin.tie(NULL); // boj_15552.cpp
#define SETPRECISION(n) cout << fixed;cout.precision(n); // boj_1008.cpp
#define SIZE(v) (int)v.size()
#define ALL(v) v.begin(),v.end()
using ll = long long;

int main() {
    FASTIO;

    string words_ignore[10] = {
        "i", "pa", "te", "ni", "niti", "a", "ali", "nego", "no", "ili"
    };

    string s, ans;
    getline(cin, s, ' ');
    ans.push_back(s[0] - 'a' + 'A');

    while (getline(cin, s, ' ')) {
        bool flag_continue = false;

        for (string w : words_ignore) {
            if (s == w) {
                flag_continue = true;
                break;
            }
        }

        if (flag_continue) continue;

        ans.push_back(s[0] - 'a' + 'A');
    }

    cout << ans << '\n';

    return 0;
}

File: baekjoon_all/03000+/boj_3181.java
// Solve 2024-02-12

// 백준에 제출할 때는 class 이름을 Main으로 설정해야 한다.

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class boj_3181 {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine(), " ");
        StringBuilder sb = new StringBuilder();
        String[] wordsIgnore = { "i", "pa", "te", "ni", "niti", "a", "ali", "nego", "no", "ili" };

        sb.append((char) (st.nextToken().charAt(0) - 'a' + 'A'));

        while (st.hasMoreTokens()) {
            String s = st.nextToken();
            boolean flagContinue = false;

            for (String w : wordsIgnore) {
                if (s.equals(w)) {
                    flagContinue = true;
                    break;
                }
            }

            if (flagContinue) continue;

            sb.append((char) (s.charAt(0) - 'a' + 'A'));
        }

        System.out.println(sb);
        br.close();
    }

}
